src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.js

```diff
@@ -48,7 +48,7 @@ export default class rd2ElevateCreditCardForm extends LightningElement {
     _paymentMethod = undefined;
     _nextDonationDate = undefined;
     _isEditPayment = false;
-    
+
```

Comment:
I didn't see any changes in this file other than whitespace. Did we mean this file to be in the PR?